tests/commands_test.py

```diff
@@ -560,7 +557,7 @@ def test_get_cmd(memcache_socket: MemcacheSocket, cache_client: CacheClient) ->
     )
     assert_called_once_with_command(
         memcache_socket.sendall,
-        b"mg lCV3WxKxtWrdY4s1+R710+9J b t l v h f R30 T300\r\n",
+        b"mg w7puw63Dp29k4o23 b t l v h f R30 T300\r\n",
```

Comment:
Any undocumented implications of no longer base64 encoding unicode keys?

Looks like this will mean a new warmup & double storing cache fights during deploys, but should be ok if all documented.

Reply:
This is still base64. The change is that now we won't generate a `blake2b` hash blindly for any unicode, only for large keys.

I also didn't like how the `Key()` class ended looking, it is best to allow bytes & any unicode and be able to encode it properly. Before it was expensive to check if something wasn't just ascii, required iterating each char, but with this rust implementation you just pass bytes, and if any char is not ascii it will b64 it and add the binary flag with minimal overhead. I very much like more this abstraction.

Regarding the compatibility, this will be a major since we break the meta-commands api completely, I'll fix docs and document this behavior change.